ConfigGenerator/MainForm.cs

```diff
@@ -222,5 +222,43 @@ internal sealed partial class MainForm : Form {
 				Tutorial.OnAction(Tutorial.EPhase.BotNicknameFinished);
 			}
 		}
-	}
+
+        private void button_update_all_configs_Click(object sender, EventArgs e)
+        {
+            GlobalConfig globalConfig = GlobalConfig.Load(Path.Combine(SharedInfo.ConfigDirectory, SharedInfo.GlobalConfigFileName));
```

Comment:
What is the purpose of this here? Did you copy-paste code without even reviewing what it does?